tasks/sf-sns-report/index.js

```diff
@@ -14,12 +15,12 @@ const cumulusMessageAdapter = require('@cumulus/cumulus-message-adapter-js');
 function eventFailed(event) {
   // event has exception
   // and it is needed to avoid flagging cases like "exception: {}" or "exception: 'none'"
-  if ((event.exception) && (typeof event.exception === 'object') &&
-    (Object.keys(event.exception).length > 0)) return true;
+  if (isObject(event.exception)
+    && (Object.keys(event.exception).length > 0)) return true;
```

Comment:
was there a preference/swap towards having `&&` on the same line to avoid any kind of `;` insertion bugs that could come up?

Reply:
@mennovandiermen pointed out that, in order to avoid automatic semicolon insertion, we should put the operators at the end of the line instead of the start.  There's a JIRA ticket written to make that change, but it's not in the scope of this ticket.  Until that ticket is completed, leading operators are what we have in our eslint config.

Reply:
Good deal, thanks for the clarification.